### PR TITLE
If Commander would goto hand, don't move it to command zone

### PR DESCRIPTION
**Pull request by tehdiplomat**

### CODE REVIEW

File: forge-ai/src/main/java/forge/ai/AiController.java [hunk 1]
```diff
@@ -1718,6 +1718,16 @@ public final boolean aiShouldRun(final ReplacementEffect effect, final SpellAbil
             hostCard = game.getCardState(hostCard);
         }
 
+        if (effect.hasParam("CommanderMoveReplacement")) {
```
Comment (tool4ever) [inline forge-ai/src/main/java/forge/ai/AiController.java:1721]: what is this for - 
it should already reach the logic in `ChangeZoneAi.doReturnCommanderLogic`?

Comment (tehdiplomat) [reply forge-ai/src/main/java/forge/ai/AiController.java:1721]: Ah i didn't know about that function. But what I'm adding is still needed. Bouncing to hand shouldn't send to Command Zone.

Comment (tool4ever) [reply forge-ai/src/main/java/forge/ai/AiController.java:1721]: yea most likely only in corner cases
might still be good to move it to the same place? 🤔

Comment (tehdiplomat) [reply forge-ai/src/main/java/forge/ai/AiController.java:1721]: Yep, I'll move it to the function you pointed out. THanks for the eyes.

**Review by tool4ever — APPROVED** [review]

**Pull request merged**